idemp-rest/WEB-INF/src/org/bandahealth/idempiere/rest/service/db/BaseInvoiceDBService.java

```diff
@@ -199,9 +209,9 @@ protected MInvoice_BH getModelInstance() {
 	 *
 	 * @return
 	 */
-	protected int getAPInvoiceDocumentTypeId() {
+	protected int getInvoiceDocumentTypeId(String baseDocType) {
```

Comment:
We can replace this method with `MDocType.getDocType(...)`.

Reply:
![doctype_query](https://user-images.githubusercontent.com/6028400/187392849-eac2a340-4bbd-4e8c-b8e9-85103ab74842.png)

For optimal purposes, would we rather get all results, load them in memory, then return the first in the list, or just query and return one result from the database? Thanks

Reply:
Since most of the time only one will be returned anyways (and, at worst, 8 will be returned), I don't think there's a problem with loading it into memory.